docs/topics/technical-operational/README.md

```diff
@@ -32,22 +32,35 @@ More information about the local EGA software and its implementation can be foun
 
 ### Standard Operating Procedures (SOPs)
 
-It is useful to establish SOPs for common node operational tasks to enable consistent service delivery and streamline internal processes. A template for writing an SOP can be found [here](https://docs.google.com/document/d/1BWPL9I9PlWiea6k-vcDDwaMu3rM5FSdV_0n8u7hAQpg/edit?usp=sharing).
+It is useful to establish SOPs for common node operational tasks to enable consistent service delivery and streamline internal processes. Use **[this template](https://docs.google.com/document/d/1BWPL9I9PlWiea6k-vcDDwaMu3rM5FSdV_0n8u7hAQpg/edit?usp=sharing)** to develop SOPs for your own node.
```

Comment:
Use this [template to develop SOPs for your own node](https://docs.google.com/document/d/1BWPL9I9PlWiea6k-vcDDwaMu3rM5FSdV_0n8u7hAQpg/edit?usp=sharing).

Reply:
better SEO